lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py

```diff
@@ -328,11 +312,9 @@ def test_basic_normal_members_predictor(self):
                      5.55444682e-01, 5.04367388e-01, 6.68575194e-01])
 
     def test_normal_mean_predictor_keyerror(self):
-        """
-        Test that the minimisation has resulted in a KeyError, if the
+        """Test that the minimisation has resulted in a KeyError, if the
```

Comment:
FYI if you look in the style guide https://github.com/metoppv/improver/wiki/Code-Style-Guide#unit-test-structure it says that unit tests should have a titile like like this
```
 """Some kind of title"""
```
not like 
```
"""
Some kind of title
"""
```

This is so it shows up as a title in the unit test output.